integration-test/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitCrossPartitionScanIntegrationTestBase.java

```diff
@@ -19,7 +19,7 @@ public abstract class ConsensusCommitCrossPartitionScanIntegrationTestBase
 
   @Override
   protected String getTestName() {
-    return "tx_cc";
```

Comment:
This string is used as namespace prefix already in https://github.com/scalar-labs/scalardb/blob/693388b4f11e816258eb6e7d0305656e213080e2/integration-test/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitIntegrationTestBase.java#L11C13-L11C18
This cause this class and `ConsensusCommitIntegrationTestWithDynamo` to have metadata and coordinator tables sharing the same name, causing failure